eal/cluster_configuration.ex

```diff
@@ -46,6 +55,9 @@ defmodule AntikytheraEal.ClusterConfiguration do
 
     @impl true
     defun zone_of_this_host() :: String.t(), do: "zone"
+
+    @impl true
+    defun health_check_grace_period() :: pos_integer, do: 400
```

Comment:
There is no reason to set the default value to 400. Maybe 300 or 0 seems to be appropriate.
https://docs.aws.amazon.com/autoscaling/ec2/userguide/healthcheck.html#health-check-grace-period

Reply:
Exactly. Because An A9a node needs a certain time, I want to use 300 as the return value.
Fixed in https://github.com/access-company/antikythera/pull/168/commits/8b5bafb82dd33b78e6180010ab3c6398cec7af95